packages/theming/default-theme/src/defaultTheme.ts

```diff
@@ -88,6 +88,65 @@ function _defaultTypography(): Typography {
   return defaultsDict;
 }
 
+export function defaultEffects(): Effects {
+  const effects =
+    Platform.OS === 'android' || Platform.OS === 'ios'
+      ? {
+          borderRadiusSmall: 2,
```

Comment:
These should probably be pulled from the global tokens in the future. Right now they don't account for mobile platforms so we need to add those.

Reply:
totally agreed.  Any opinions on how/if I should handle this in the interim?

Reply:
I think as it is is fine. The global tokens we have in FURN have the win32 tokens available if you'd like to use them, but otherwise I'll come back and fix it later.